src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Sami32 Here's the 10 bit/200 Mb/s with hardware acceleration. It manage to creep above 1x in the end, but it was in the 0.9x'es for a while so for that playback to be successful I would have needed a solid buffer. In any case, there are so many different hardware configurations out there, and I can guarantee you that a lot of the computers people use (pretty much any laptop) is weaker than this computer, so you can't really expect these results.

```Powershell
ffmpeg.exe -y -hwaccel auto -i jellyfish-200-mbps-4k-uhd-hevc-10bit.mkv -c:v h264_nvenc -tune fastdecode -crf 22 -c:a aac -b:a 320k -ar 48000 -pix_fmt yuv420p -f ismv -movflags isml+frag_keyframe hevc_test1.mp4 -benchmark
ffmpeg version N-86986-g62dfa2b Copyright (c) 2000-2017 the FFmpeg developers
  built with gcc 7.1.0 (GCC)
  configuration: --enable-gpl --enable-version3 --enable-cuda --enable-cuvid --enable-d3d11va --enable-dxva2 --enable-libmfx --enable-nvenc --enable-avisynth --
enable-bzlib --enable-fontconfig --enable-frei0r --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-li
bfreetype --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enabl
e-libopenh264 --enable-libopenjpeg --enable-libopus --enable-librtmp --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolam
e --enable-libvidstab --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable
-libxavs --enable-libxvid --enable-libzimg --enable-lzma --enable-zlib
  libavutil      55. 74.100 / 55. 74.100
  libavcodec     57.102.100 / 57.102.100
  libavformat    57. 76.100 / 57. 76.100
  libavdevice    57.  7.100 / 57.  7.100
  libavfilter     6. 99.100 /  6. 99.100
  libswscale      4.  7.102 /  4.  7.102
  libswresample   2.  8.100 /  2.  8.100
  libpostproc    54.  6.100 / 54.  6.100
Input #0, matroska,webm, from 'jellyfish-200-mbps-4k-uhd-hevc-10bit.mkv':
  Metadata:
    COMPATIBLE_BRANDS: iso4hvc1iso6
    MAJOR_BRAND     : iso4
    MINOR_VERSION   : 1
    ENCODER         : Lavf56.3.100
  Duration: 00:00:30.10, start: 0.067000, bitrate: 199827 kb/s
    Stream #0:0(und): Video: hevc (Main 10), yuv420p10le(tv), 3840x2160 [SAR 1:1 DAR 16:9], 29.97 fps, 29.97 tbr, 1k tbn, 29.97 tbc (default)
    Metadata:
      CREATION_TIME   : 2016-02-05 13:43:30
      LANGUAGE        : und
      HANDLER_NAME    : hevc@GPAC0.5.2-DEV-rev565-g71748d7-ab-suite
Codec AVOption tune (Tune the encoding to a specific scenario) specified for output file #0 (hevc_test1.mp4) has not been used for any stream. The most likely r
eason is either wrong type (e.g. a video option with no video streams) or that it is a private option of some encoder which was not actually used for any stream
.
Codec AVOption crf (Select the quality for constant quality mode) specified for output file #0 (hevc_test1.mp4) has not been used for any stream. The most likel
y reason is either wrong type (e.g. a video option with no video streams) or that it is a private option of some encoder which was not actually used for any str
eam.
Codec AVOption b (set bitrate (in bits/s)) specified for output file #0 (hevc_test1.mp4) has not been used for any stream. The most likely reason is either wron
g type (e.g. a video option with no video streams) or that it is a private option of some encoder which was not actually used for any stream.
Stream mapping:
  Stream #0:0 -> #0:0 (hevc (native) -> h264 (h264_nvenc))
Press [q] to stop, [?] for help
[hevc @ 000000000036a5a0] Either a hw_frames_ctx or a hw_device_ctx needs to be set for hardware decoding.
Output #0, ismv, to 'hevc_test1.mp4':
  Metadata:
    COMPATIBLE_BRANDS: iso4hvc1iso6
    MAJOR_BRAND     : iso4
    MINOR_VERSION   : 1
    encoder         : Lavf57.76.100
    Stream #0:0(und): Video: h264 (h264_nvenc) (Main) (avc1 / 0x31637661), yuv420p, 3840x2160 [SAR 1:1 DAR 16:9], q=-1--1, 2000 kb/s, 29.97 fps, 10000k tbn, 29.
97 tbc (default)
    Metadata:
      CREATION_TIME   : 2016-02-05 13:43:30
      LANGUAGE        : und
      HANDLER_NAME    : hevc@GPAC0.5.2-DEV-rev565-g71748d7-ab-suite
      encoder         : Lavc57.102.100 h264_nvenc
    Side data:
      cpb: bitrate max/min/avg: 0/0/2000000 buffer size: 4000000 vbv_delay: -1
frame=  900 fps= 34 q=46.0 Lsize=    7968kB time=00:00:29.99 bitrate=2176.1kbits/s speed=1.13x
video:7959kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.115596%
bench: utime=274.000s
bench: maxrss=1086668kB
```